### PR TITLE
Add diagram "DID document entries".

### PR DESCRIPTION
**Pull request by peacekeeper**
For the "Data Model" section:

_Diagram illustrating the entries in the DID document, including properties and representation-specific entries; some entries are defined by this specifications; others are defined by registered or unregistered extensions._

diagram-did-document-entries

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/595.html" title="Last updated on Feb 4, 2021, 9:56 AM UTC (8baa27b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/595/6f647ff...8baa27b.html" title="Last updated on Feb 4, 2021, 9:56 AM UTC (8baa27b)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1133,6 +1133,18 @@ <h1>Data Model</h1>
 The second class are representation-specific entries, and
 are specified in section <a href="#representations"></a>.
     </p>
+
+    <figure id="did-document-entries">
+      <img style="margin: auto; display: block;"
+        src="diagrams/diagram-did-document-entries.png" alt="
+Diagram illustrating the entries in the DID document, including properties
+and representation-specific entries; some entries are defined by this
+specifications; others are defined by registered or unregistered extensions.">
```
Comment (TallTed) [inline index.html:1142]:
```suggestion
specification; others are defined by registered or unregistered extensions.">
```

**Comment by TallTed** [comment]
Comments by @msporny at https://github.com/w3c/did-core/pull/460#discussion_r567290301 apply here as well -- i.e., see "our accessibility requirements: #118 -- the description should go in a longdesc attribute associated with each image so that people using screen readers will be able to hear the descriptive text while those that can rely on their sense of sight will be able to look at the image."

**Comment by iherman** [comment]
> Comments by @msporny at [#460 (comment)](https://github.com/w3c/did-core/pull/460#discussion_r567290301) apply here as well -- i.e., see "our accessibility requirements: #118 -- the description should go in a longdesc attribute associated with each image so that people using screen readers will be able to hear the descriptive text while those that can rely on their sense of sight will be able to look at the image."

It is not quite clear to me what the status of `longdesc` is. It is still a bona fide attribute in HTML5 but, for example, MDN notes it as obsolete. One possible replacement, I believe, is to use put an explicit link to the figcaption element (or use the details element).

@michael-n-cooper can you chime in on this, please?

**Review by OR13 — APPROVED** [review]
@peacekeeper this diagram shows `@context` as being present in the infra map. I agree with you that it might not survive the translation to JSON or CBOR, depending on the DID Methods support for representations.

**Comment by rhiaro** [comment]
For #118 this is better as an SVG with actual text in, not an image of text. @iherman may be able to help with that?

**Comment by peacekeeper** [comment]
> @peacekeeper this diagram shows `@context` as being present in the infra map.

Personally I still believe it is a mistake to include anything that's representation-specific in the abstract data model, but that was the preference of the WG, so that's what the diagram shows. :)

**Comment by iherman** [comment]
> For #118 this is better as an SVG with actual text in, not an image of text. @iherman may be able to help with that?

True, but it is delicate (alas!). SVG does not have a feature to align the text's size automatically so that it would fit a rectangle. If the font used is not present on the environment where SVG displays, then the text may overflow, and make the diagram unusable. If a common font is used, like Helvetica or Times, then the chances of having that font on everyone's machine is of course big. But the probability for failure is non-zero. That is why many authoring systems turn characters into geometry before dumping an SVG version. It is not great, to say the least, for accessibility reasons. One possibility is to use additional titles in the diagram, but it is a drag.

Maybe what we should do is to produce diagrams with a common font, use them as text in SVG and we all test this on our respective machines, including iPhones, android phones, tablets, etc. With such a techie crowd we would probably cover most of the machine types out there.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**